src/lib/app.js

```diff
@@ -8,14 +8,16 @@
 
 import './webcomponents-config'; // must go before -loader below
 import '@webcomponents/webcomponentsjs/webcomponents-loader.js';
-import {trackError} from './analytics'; // side effects & named export
 import {swapContent, getHTML} from './loader';
 import * as router from './utils/router';
 import {checkUserPreferredLanguage} from './actions';
 import {store} from './store';
 import {localStorage} from './utils/storage';
 import removeServiceWorkers from './utils/sw-remove';
-import {syncContentIndex} from './content-indexing';
+// TODO: Enable this when #3836 is fixed.
+// // https://github.com/GoogleChrome/web.dev/issues/3836
+// import {trackError} from './analytics'; // side effects & named export
```

Comment:
I think you want to keep this? Or at least double-check what the side-effects of the import is. I remember that the `./analytics` import was around before my PR.

Reply:
It looks like it's ok being commented out because your snippet on line 70 was the only one using it.

Reply:
OH WAIT you're right

Reply:
didn't realize that module had its own side effects. Good catch.